### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by kritzikratzi** [rickbrew/GeneralPolygonClipper#2] [Q&A] [answered]
You have placed this project under the MIT license, but both the readme and the source code contain this phrasing: 

	This software is free for non-commercial use. It may be copied,
	modified, and redistributed provided that this copyright notice
	is preserved on all copies. The intellectual property rights of
	the algorithms used reside with the University of Manchester
	Advanced Interfaces Group.
	You may not use this software, in whole or in part, in support
	of any commercial product without the express consent of the
	author.
	There is no warranty or other guarantee of fitness of this
	software for any purpose. It is provided solely "as is".

IANAL, but i'm pretty sure that's incompatible to the MIT license. My suggestion is to remove everything from the current license file, and put the above paragraph instead. 

Particularly this line in the current file seems problematic: 

    Copyright (c) 2021 Rick Brewster

I'm sure you mean it in good faith, and maybe github auto-generated the license file for you. The source code clearly states `Alan Murta (email: gpc@cs.man.ac.uk)` as author.

**Comment by rickbrew** [comment, marked as answer]
Ahh yes, I believe you're right. It was just an auto-generated license file that I didn't look at. However, Alan Murta did contact me on Twitter and was fine with the MIT License. I'll be sure to get the author credit and copyright fixed though.

**Comment by rickbrew** [comment]
I also updated the code to version 2.33.

**Comment by rickbrew** [comment]
Also, here's the Twitter message I received from Alan Murta
<img width="488" alt="image" src="https://user-images.githubusercontent.com/10873410/147713426-74f28af4-f4f0-42c8-893d-ac0863523f7b.png">

**Comment by rickbrew** [comment]
If you see any other issues, please don't hesitate to post again.

**Comment by kritzikratzi** [comment]
allowing the change from non-commercial-evaluation-only to MIT is incredibly generous. i'm glad it worked out this way. thanks for contacting him and figuring things out!
